packages/pkh-tezos/src/authmethod.ts

```diff
@@ -0,0 +1,99 @@
+import { AccountId } from 'caip'
+import { randomString } from '@stablelib/random'
+import { Cacao, SiwTezosMessage, AuthMethod, AuthMethodOpts } from '@didtools/cacao'
+
+export const TEZOS_MAINNET_CHAIN_REF = 'NetXdQprcVkpaWU' // Tezos mainnet beta
+export const TEZOS_DEVNET_CHAIN_REF = 'NetXm8tYqnMWky1' // Tezos devnet beta
+export const VERSION = '1'
+export const CHAIN_NAMESPACE = 'tezos'
+
+export const chainIdMap = {
+  mainnet: TEZOS_MAINNET_CHAIN_REF,
+  devnet: TEZOS_DEVNET_CHAIN_REF,
+}
+
+type TezosNetwork = 'mainnet' | 'devnet'
+
+export namespace TezosWebAuth {
+  // eslint-disable-next-line @typescript-eslint/require-await
+  export async function getAuthMethod(
+    tzProvider: any,
+    account: AccountId,
+    publicKey: string
+  ): Promise<AuthMethod> {
+    if (typeof window === 'undefined')
+      throw new Error('Web Auth method requires browser environment')
+    const domain = (window as Window).location.hostname
+
+    return async (opts: AuthMethodOpts): Promise<Cacao> => {
+      opts.domain = domain
+      return createCACAO(opts, tzProvider, account, publicKey)
+    }
+  }
+}
+
+export type SupportedProvider = {
+  requestSignPayload: (opts: {
+    signingType: string
+    payload: string
+  }) => Promise<{ signature: string }>
+}
+
+export function assertSupportedProvider(tzProvider: any): asserts tzProvider is SupportedProvider {
+  const p = tzProvider as SupportedProvider
+  if (p.requestSignPayload == null) {
+    throw new Error('Unsupported provider; provider must implement requestSignPayload')
+  }
+}
+
+async function sign(tzProvider: any, message: string) {
+  assertSupportedProvider(tzProvider)
+  const { signature } = await tzProvider.requestSignPayload({
+    signingType: 'micheline',
+    payload: message,
+  })
+  return signature
+}
+
+async function createCACAO(
+  opts: AuthMethodOpts,
+  tzProvider: any,
+  account: AccountId,
+  publicKey: string
+): Promise<Cacao> {
+  const now = new Date()
+  const oneDayLater = new Date(now.getTime() + 24 * 60 * 60 * 1000)
```

Comment:
we changed to 1 week in the other libraries now, if you can do the same here 
```suggestion
  const now = new Date()
  const oneWeekLater = new Date(now.getTime() + & * 24 * 60 * 60 * 1000)
```

Reply:
Fixed.